pkg/controller/kieapp/constants/constants.go

```diff
@@ -272,6 +272,8 @@ const (
 
 	//Kafka
 	KafkaTopicsEnv = "KIE_SERVER_KAFKA_EXT_TOPICS"
+
+	ACFilters = "AC_ALLOW_ORIGIN,AC_ALLOW_METHODS,AC_ALLOW_HEADERS,AC_ALLOW_CREDENTIALS,AC_MAX_AGE"
```

Comment:
Please add comment to the code for what are these filters, right now it may look like something used by Kafka. e.g. just `//CORS`

Reply:
ok